react/features/base/media/components/native/Audio.js

```diff
@@ -44,9 +44,13 @@ export default class Audio extends AbstractAudio {
         this._sound
             = this.props.src
                 ? new Sound(
-                    this.props.src,
+                    this.props.src, null,
```

Comment:
So I did not want to limit the ability to play sounds from other protocols. By specifying base path there it would not allow to play from file: or http:. I'm not sure how this would make it more cross platform, but I agree that would be more convenient.

There was an idea to keep base/media/Audio element as generic as possible and as close as possible to HTMLAudioElement, so limiting it to play only "asset" sounds would be a step in the opposite direction.

Reply:
Btw. the null argument is required with the current version of 'react-native-sounds'. Because it has a bug or something where it tries to play from a string which equals to src + basePath which is a callback converted to string and ends up as something weird.

Reply:
I see, disregard my comment then!